soql-environment/src/main/scala/com/socrata/soql/environment/TableName.scala

```diff
@@ -29,6 +29,7 @@ object TableName {
   val SoqlPrefix = "@"
   val PrimaryTable = TableName(SodaFountainPrefix)
   val This = SodaFountainPrefix + "this"
+  val SingleRow = SodaFountainPrefix + "single_row"
```

Comment:
Thought: Also do `val ReservedNames = [This, SingleRow]`, and then we'd be able to throw exceptions if someone tries to create an AnalysisContext using these?

Reply:
Probably, yeah, actually....